vendor/k8s.io/kubernetes/pkg/apis/core/validation/validation.go

```diff
@@ -1773,27 +1773,26 @@ func ValidatePersistentVolumeClaimSpec(spec *core.PersistentVolumeClaimSpec, fld
 func ValidatePersistentVolumeClaimUpdate(newPvc, oldPvc *core.PersistentVolumeClaim) field.ErrorList {
 	allErrs := ValidateObjectMetaUpdate(&newPvc.ObjectMeta, &oldPvc.ObjectMeta, field.NewPath("metadata"))
 	allErrs = append(allErrs, ValidatePersistentVolumeClaim(newPvc)...)
+	newPvcClone := newPvc.DeepCopy()
```

Comment:
fix lgtm. Would even add a big red warning sign and a 😱 .

Reply:
nit: do not even call them `*Clone`, better shadow them. Avoid that somebody chooses the wrong PVC by accident.

Reply:
ack, I think that is sensible I wrote upstream code. :((